docs/framework/upgrade.md

```diff
@@ -81,5 +81,4 @@ All Smarty tags are evaluated before the sql is run. Commonly used Smarty variab
 
 Testing When testing upgrades, it's important to run the upgrades against several different databases (representing different versions of CiviCRM and different upgrade paths). The tool "civicrm-upgrade-test" can automate this process. For instructions:
 
-* If you are using buildkit you can use the integrated [civibuild upgrade-test](https://github.com/civicrm/civicrm-buildkit/blob/master/README.md#daily-coding-upgrade-testing) command.
```

Comment:
Aww, did we lose the tutorial for the upgrade-testing workflow? Maybe it should migrated into this doc?

https://github.com/civicrm/civicrm-buildkit/blob/37abb2e6dd9790ffc016377dc2e20c0965e2953d/doc/daily-coding.md#upgrade-tests